src/psyclone/tests/domain/gocean/transformations/gocean_opencl_trans_test.py

```diff
@@ -328,7 +379,7 @@ def test_opencl_routines_initialisation(kernel_outputdir):
     for kernel in sched.coded_kernels():
         trans.apply(kernel)
 
```

Comment:
My pylint says that the 'disable' at line 371 is "useless".

Reply:
Removed